.github/workflows/publish-test.yaml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package to Test PyPi
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Do you need to install pip? That should come by default with python 3

Reply:
I actually took this from the github action template it suggested and edited for different secrets, but I have also just seen this: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ so I might redo it following this?

Reply:
I think they use the same action

Reply:
oh nice!

Reply:
Absolutely useless documentation.
`In this example, we’ll use pep517 package, assuming that your project has a pyproject.toml properly set up (see PEP 517/PEP 518).`

and then they don't give you any help on what a proper `pyproject.toml` is or looks like.